examples/ComponentHosting/Podfile

```diff
@@ -0,0 +1,6 @@
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '7.0'
+target 'Sample' do
+  pod 'ComponentKit', :git => 'https://github.com/facebook/componentkit.git', :commit => '115d2012580f17276b5383cd1dad2b306b24aeb1'
```

Comment:
Yeah, master on ComponentKit has a different API than the latest tag (0.14). I'll try to get them to push out a new tag.